### PR TITLE
Specify result type of `ConstraintSchemaInterface::getTableIndexes()` method to `IndexConstraint[]`

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/Schema/AbstractSchema.php [hunk 1]
```diff
@@ -261,7 +262,7 @@ public function getTableForeignKeys(string $name, bool $refresh = false): array
      */
     public function getTableIndexes(string $name, bool $refresh = false): array
     {
-        /** @psalm-var mixed $tableIndexes */
+        /** @var IndexConstraint[]|null $tableIndexes */
```
Comment (Tigrov) [inline src/Schema/AbstractSchema.php:265]: Are there cases when it returns `null`?

Comment (vjik) [reply src/Schema/AbstractSchema.php:265]: Hm... I don't know. If no, then we can remove check "is_array".

Comment (Tigrov) [reply src/Schema/AbstractSchema.php:265]: > Hm... I don't know. If no, then we can remove check "is_array".

Yes, looks like this.

Comment (vjik) [reply src/Schema/AbstractSchema.php]: Done

**Review by Tigrov — APPROVED** [review]

**Pull request merged**